src/bun.js/test/jest.zig

```diff
@@ -493,8 +493,7 @@ pub const Jest = struct {
         var filepath = Fs.FileSystem.instance.filename_store.append([]const u8, slice) catch unreachable;
 
         var scope = runner_.getOrPutFile(filepath);
-        DescribeScope.active = scope;
-        DescribeScope.module = scope;
+        scope.push();
```

Comment:
Two questions:
- This continues to work with `--preload` adding global before{all,Each} and after{all,Each} which are cross-module, right?
- Cross-module before{all,Each} and after{all,Each} continue to not run after the file has finished executing, right?

Reply:
> Cross-module before{all,Each} and after{all,Each} continue to not run after the file has finished executing, right?

Do you mean if `bun test` is run across multiple files, `beforeAll` etc. from the first file won't get run when processing subsequent files?

If so, I can confirm that is indeed the case via stepping through with the debugger.